lms/djangoapps/teams/static/teams/templates/team-profile.underscore

```diff
@@ -26,15 +26,15 @@
 
         <div class="team-capacity">
             <span class="sr"><%- gettext("Team capacity") %></span>
-            <span><%- membershipText %></span>
+            <span><%= HtmlUtils.ensureHtml(membershipText) %></span>
```

Comment:
This change doesn't seem necessary. ``<%- membershipText %>`` will correctly escape the text.